packages/cli-test/src/cli/commands/platform.spec.ts

```diff
@@ -96,10 +96,12 @@ describe('platform commands', () => {
       sandbox.stub(shell, 'kill').rejects();
       await assert.rejects(platform.runStop({ proc: fakeProcess }));
     });
-    it('should reject if waitForShutdown=true and waitForOutput rejects', async () => {
-      sandbox.stub(shell, 'kill').resolves();
-      waitForOutputSpy.rejects();
-      await assert.rejects(platform.runStop({ proc: fakeProcess, waitForShutdown: true }));
+    it('non-Windows only: should reject if waitForShutdown=true and waitForOutput rejects', async () => {
+      if (process.platform !== 'win32') {
```

Comment:
The reason we don't run this test on Windows is that [this code-under-test also doesn't run on Windows](https://github.com/slackapi/node-slack-sdk/blob/main/packages/cli-test/src/cli/commands/platform.ts#L149-L153)